packages/exchange/src/adapters/encryptus/api.test.ts

```diff
@@ -130,3 +131,22 @@ describe('getEncryptusBaseUrl', () => {
     expect(getCryptoAddress).toBeDefined()
   })
 })
+
+describe('encryptusExtractParamsFromBaseUrl', () => {
+  it('extracts the params from the base url', () => {
+    const baseUrl = `https://sandbox.encryptus.co/pw?access_token=FAKE_TOKEN`
+    const params = encryptusExtractParamsFromBaseUrl(baseUrl.toString())
+    expect(params).toEqual({access_token: 'FAKE_TOKEN'})
```

Comment:
this validation should be included in the API return -> after parsing the schema, if the schema is correct, then check if the data is correct.

Reply:
ok i will simplify this

Reply:
it can throw Api.Error.MalformatedResponse